cms/templates/settings.html

```diff
@@ -347,7 +352,11 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                     </span>
 
                     <span class="msg msg-help">
-                    ${_("You can manage this image along with all of your other <a href='{}'>files &amp; uploads</a>").format(upload_asset_url)}
+                    ${Text(_("You can manage this image along with all of your other {files_uploads_html}")).format(
+                        files_uploads_html=HTML("<a href='{upload_asset_url}'>files & uploads</a>").format(
```

Comment:
The `&` needs to stay `&amp;` because it is directly inside `HTML()` and won't be escaped.  In this case, I think the linter wouldn't have reported this as an issue.  So, with actual strings, you can lean on the linter more heavily...unless we find some bugs. ;)